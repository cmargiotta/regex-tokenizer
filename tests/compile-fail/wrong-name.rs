use regex_tokenizer::tokenizer;

fn main() {
    tokenizer! {
        123
    }
}
