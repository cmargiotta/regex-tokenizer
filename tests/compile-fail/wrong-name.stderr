error: Unexpected token, an identifier is needed
 --> tests/compile-fail/wrong-name.rs:5:9
  |
5 |         123
  |         ^^^
